src/geode/mesh/builder/polygonal_surface_view_builder.cpp

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2019 - 2020 Geode-solutions
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ *
+ */
+
+#include <geode/mesh/builder/polygonal_surface_view_builder.h>
+
+#include <geode/mesh/core/polygonal_surface_view.h>
+
+namespace geode
+{
+    template < index_t dimension >
+    void PolygonalSurfaceViewBuilder< dimension >::do_set_mesh(
+        VertexSet& mesh )
+    {
+        polygonal_surface_view_ =
+            &dynamic_cast< PolygonalSurfaceView< dimension >& >( mesh );
+    }
+
+    template < index_t dimension >
+    void PolygonalSurfaceViewBuilder< dimension >::do_set_point(
+        index_t /*unused*/, const Point< dimension >& /*unused*/ )
+    {
+        // polygonal_surface_view_->set_vertex( vertex_id, point, {} );

```

Comment:
to remove? or to prepare the future?

Reply:
API is requested to satify inheritance. Implementation will be added later.